server/app/services/applicant/ApplicantData.java

```diff
@@ -149,4 +149,15 @@ public ImmutableMap<Path, String> getFailedUpdates() {
   public boolean updateDidFailAt(Path path) {
     return getFailedUpdates().containsKey(path);
   }
+
+  public Optional<String> getDOB() {
+    String dob = readString(WellKnownPaths.APPLICANT_DOB).get();
+    return Optional.of(dob);
+  }
+
+  public void setDOB(@Nullable String dob) {
+    if (!hasPath(WellKnownPaths.APPLICANT_DOB)) {
```

Comment:
Not blocking: The current behavior silently doesn't update if the DOB has already been set. That behavior seems surprising to me. Would it be acceptable to instead throw some sort of error indicating that the DOB can't be set multiple times?